managed/services/agents/jobs.go

```diff
@@ -477,7 +484,7 @@ func (s *JobsService) StartMongoDBRestoreBackupJob(
 	locationConfig *models.BackupLocationConfig,
 ) error {
 	if err := PMMAgentSupported(s.r.db.Querier, pmmAgentID,
-		"mongodb restore", pmmAgentMinVersionForMongoDBBackupAndRestore); err != nil {
+		"mongodb restore", pmmAgentMinVersionForMongoLogicalBackupAndRestore); err != nil {
```

Comment:
restore is currently out of scope for this so this will be cared for in a separate ticket.